8-Reinforcement/README.md

```diff
@@ -1,4 +1,4 @@
-# Introduction to reinforcement learning
+ Introduction to reinforcement learning
```

Comment:
I think this should be reverted?

Reply:
Sorry but I only worked on part 7 and 9

Reply:
I understand but you might have removed inadvertently the "#" before the title on your branch. Otherwise you wouldn't have this file modified in your pull request.

Reply:
oh yes I see it now sorry